pulser-core/pulser/json/abstract_repr/deserializer.py

```diff
@@ -292,6 +293,10 @@ def _deserialize_channel(obj: dict[str, Any]) -> Channel:
         if obj["eom_config"] is not None:
             data = obj["eom_config"]
             try:
+                optional_keys = ("multiple_beam_control",)
```

Comment:
Won't in be better to have `optional_keys` as a constant variable outside the function ? Like `OPTIONAL_EOM_KEYS = ("multiple_beam_control",)`